conduct/artifacts/CODE_OF_CONDUCT_PATCH.patch

```diff
@@ -29,29 +20,3 @@
  All complaints will be reviewed and investigated promptly and fairly.
  
  All community leaders are obligated to respect the privacy and security of the
-@@ -155,13 +157,13 @@
- individual, or aggression toward or disparagement of classes of individuals.
- 
- **Consequence**: A permanent ban from any sort of public interaction within the
--community.
-+project community.
```

Comment:
Dropping this diff between our version and the upstream is the core reason why the earlier conversation on updating to 2.1 stalled, and it should not pass here without comment or question.

The meaning of "community" needs to be clarified. If a permanent ban is applied to an individual, are they:
1. Banned from interacting within the community of the project in which they've misbehaved, or
2. Banned from interacting within any and all OpenJS project communities?

My preferred option here would be to include some text in the preamble clarifying the term and its interpretation.

Our [Contributor License Agreement](https://individual-cla.openjsf.org/) refers to the whole of the foundation as one "Project", in order to keep the base Apache CLA unmodified. I mention this because in addition to the CoC, as far as I know that's the one other document that we're asking community members to accede to.

Reply:
Clarifying it sounds great. The Foundation afaik doesn't and imo shouldn't have the ability to ban anyone across all OpenJS projects - that's up to each project.